src/plugins/saved_objects_management/public/services/service_registry.ts

```diff
@@ -32,7 +32,7 @@ import { SavedObjectLoader } from '../../../saved_objects/public';
 
 export interface SavedObjectsManagementServiceRegistryEntry {
   id: string;
-  service: SavedObjectLoader;
+  service: SavedObjectLoader | any;
```

Comment:
this is needed since we also need to access non-saved-object-loader services here. this include uiActions.